queue/src/main/java/com/bazaarvoice/emodb/queue/core/stepfn/StepFunctionService.java

```diff
@@ -0,0 +1,76 @@
+package com.bazaarvoice.emodb.queue.core.stepfn;
+
+
+import com.amazonaws.services.stepfunctions.AWSStepFunctions;
+import com.amazonaws.services.stepfunctions.AWSStepFunctionsClientBuilder;
+import com.amazonaws.services.stepfunctions.model.StartExecutionRequest;
+import com.amazonaws.services.stepfunctions.model.StartExecutionResult;
+import com.amazonaws.services.stepfunctions.model.StateMachineDoesNotExistException;
+import com.amazonaws.services.stepfunctions.model.InvalidArnException;
+import com.amazonaws.services.stepfunctions.model.InvalidExecutionInputException;
+import com.amazonaws.services.stepfunctions.model.AWSStepFunctionsException;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+/**
+ * Production-level service to interact with AWS Step Functions using AWS SDK v1.
+ */
+public class StepFunctionService {
+
+    private static final Logger logger = LoggerFactory.getLogger(StepFunctionService.class);
+
+    private final AWSStepFunctions stepFunctionsClient;
+
+    /**
+     * Constructor to initialize Step Function Client with AWS region and credentials.
+     */
+    public StepFunctionService() {
+        this.stepFunctionsClient = AWSStepFunctionsClientBuilder.standard()
+                .withRegion("us-east-1")
```

Comment:
Read from environment variables / config file.

Reply:
+1, surely we will have step functions in each region right?

Reply:
yes Igor, this he will update the same pr.